### ISSUE TITLE
Add CMYK color format

### ISSUE THREAD

**Issue by lukebsa** [sindresorhus/System-Color-Picker#28]
Can you add the option to use CMYK color format? Great app!

**Comment by sindresorhus** [comment]
Can you elaborate? There's already CMYK support on the second tab in the app.

Or do you mean CMYK support for the text inputs. If so, what format would you expect the CMYK to be outputted and written in? For example, HSL uses the format `hsl(0deg 0% 100%)`, but there's no convention for CMYK.

**Comment by lukebsa** [comment]
I mean CMYK support for text inputs. I would suggest percentage `CMYK(0%, 89%, 76%, 11%)` or decimal `CMYK(0.00, 0.89, 0.76, 0.11)`. Perhaps this could be changed in the config.

**Comment by sindresorhus** [comment]
Yeah, sounds good. `CMYK(` => `cmyk` though. I'll look into this in a month or so.

**Comment by bhanquier** [comment]
I fully support the request, and awaiting this addition to this awesome app.
Of lukebsa's suggestions for format I think percentage is preferable as it seems to me it's used more often in apps in general.
Cheers!

**Comment by sindresorhus** [comment]
CMYK is now supported in the latest update (2.2.0).
